C7/Game.cs

```diff
@@ -412,7 +412,14 @@ public override void _UnhandledInput(InputEvent @event)
 				using (var gameDataAccess = new UIGameDataAccess()) {
 					var tile = mapView.tileOnScreenAt(gameDataAccess.gameData.map, eventMouseButton.Position);
 					if (tile != null) {
+						bool shiftDown = Input.IsKeyPressed((int)Godot.KeyList.Shift);
+						if (shiftDown && tile.cityAtTile != null && tile.cityAtTile.owner == controller)
```

Comment:
I believe `tile.cityAtTile != null && tile.cityAtTile.owner == controller` can also be evaluated like `tile.cityAtTile?.owner == controller`

Reply:
It can be though they're not exactly the same. Rewriting it that way, it will pass if cityAtTile and controller are both null. That shouldn't matter right now since I don't think it's possible for controller to be null, but maybe some day it will be like for an observer player. Speaking of some day, we might want to add an empty city object and use that instead of null for cityAtTile. Anyway, in the mean time I'll replace it.